ts/indicators/ichimoku-kinko-hyo.src.ts

```diff
@@ -0,0 +1,1009 @@
+/* *
+ *
+ *  License: www.highcharts.com/license
+ *
+ * */
+
+'use strict';
+
+import H from '../parts/Globals.js';
+
+/* eslint-disable @typescript-eslint/interface-name-prefix */
+
+/**
+ * Internal types
+ * @private
+ */
+declare global {
+    namespace Highcharts {
+
+        class IKHIndicator
+            extends SMAIndicator {
+            public data: Array<IKHIndicatorPoint>;
+            public graphCollection: Array<string>;
+            public graphsenkouSpan: (SVGElement | undefined)
+            public ikhMap: Dictionary<Array<IKHIndicatorPoint>>;
+            public nextPoints: Array<IKHIndicatorPoint>;
+            public options: IKHIndicatorOptions;
+            public pointArrayMap: Array<keyof IKHIndicatorPoint>;
+            public pointValKey: string;
+            public pointClass: typeof IKHIndicatorPoint;
+            public points: Array<IKHIndicatorPoint>;
+            public drawGraph(): void;
+            public init(): void;
+            public toYData(point: Point): Array<number>;
+            public translate(): void;
+
+        }
+
+        interface IKHIndicatorDrawSenkouSpanOptions {
```

Comment:
These are only arguments for a function, so the naming should be end on `Object`.

Reply:
This is not solved yet.

Reply:
Sorry, I could have swore that I renamed it. Corrected the name now.